src/test/java/hudson/scm/AbstractSubversionTest.java

```diff
@@ -57,12 +67,9 @@ public static Proc runSvnServe(URL zip) throws Exception {
      * Runs svnserve to serve the specified directory as a subversion repository.
      */
     public static Proc runSvnServe(File repo) throws Exception {
```

Comment:
Probably this utility could be better moved to `SubversionSampleRepoRule` as well, but leaving for another day.